### PR TITLE
Better support for Python models

### LINKED ISSUE #294: [CT-942] Better docs support for Python models
Follow-up to #292

- Add model language to table details
- Syntax highlighting for Python + SQL
- Add python model to CI project
- ...?

### Timeline
- Out of scope for first beta
- In scope for subsequent beta / RC

### PR DESCRIPTION
**Pull request by jtcohen6**
resolves #294

- [x] Add model language to table details
- [x] Syntax highlighting for Python + SQL
- [x] Add python model to CI project

Screenshot 2022-07-27 at 13 57 48

### CODE REVIEW

File: src/app/components/code_block/code_block.html [hunk 1]
```diff
@@ -14,8 +14,9 @@ <h6>Code</h6>
             </li>
         </ul>
         <div style="margin-top: 1px">
+            <!--- TODO: dynamically update "language-sql" or "language-python" based on model scope --->
             <pre style="background-color: white"
-                 class='code line-numbers'><code class="source-code highlight language-sql" ng-bind-html="highlighted"></code></pre>
+                 class='code line-numbers'><code class="source-code highlight language-python" ng-bind-html="highlighted"></code></pre>
```
Comment (jtcohen6) [inline src/app/components/code_block/code_block.html:19]: @drewbanin Any chance I could ask for your help here? :)

Big idea:
- If `mod.language == "python"`, we want this to be `language-python`
- If `mod.language == "sql"` (or undefined), we want this to be `language-sql`

I'm not enough of an Angular pro to know the ins and outs of `ng-class` & friends

Comment (drewbanin) [reply src/app/components/code_block/code_block.html:19]: I think you can do
```
<code
    class="source-code highlight" ng-class="mod.language == 'python' ? 'language-python' : 'language-sql'"
    ng-bind-html="highlighted">
</code>
```

or, you could create [a scope variable](https://github.com/dbt-labs/dbt-docs/blob/main/src/app/components/code_block/code_block.js#L19-L20) in the component called `language_class` (or w/e) with a value of `language-sql` / `language-python` and then write:
```
<code class="source-code highlight" ng-class="language_class" ng-bind-html="highlighted">
</code>
```

Comment (jtcohen6) [reply src/app/components/code_block/code_block.html:19]: Thank you! I tried things sorta in this vein (random guesses), but I think the issue is that `mod`/`mod.language` isn't available in the `codeBlock` directive / in scope for `code_block.html`

Screenshot 2022-07-27 at 15 01 14

**Review by ChenyuLInx — APPROVED** [review]
Looks great! Thanks for adding this!

**Comment by jtcohen6** [comment]
Merge plan:
- other PR branch first
- then we can merge that PR with all the changes
- build + distribute to `dbt-core`

**Pull request merged**